src/GraphQL.Relay/GraphQL.Relay.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphQL" Version="5.2.0" />
+    <PackageReference Include="GraphQL" Version="$(GraphQLTestVersion)" />
```

Comment:
This is incorrect. This should be the minimum version supported by the project. This is the way it will be built when packaged. 

The tests should run with the test project having a reference to GraphQL using GraphQLVersion to simulate having a newer version installed than the project was built for.

Reply:
Please see the GraphQL.Authorization repo for a properly configured sample

Reply:
> This is incorrect. This should be the minimum version supported by the project. This is the way it will be built when packaged.

For "normal" project yes. For 0.x.x projects as one here my intent the opposite - update to the latest available for 0.7 with all bugfixes and features available.

Reply:
I moved property.

Reply:
I understand and I'm okay if we have the "minimum" version as 5.3.1 in this case as an 0.x "beta" version.  I just want to make sure that the build process is set up correctly.  All looks good now.  Feel free to bump the minimum version up if you want, or leave it as-is.